src/watchmaker/static/shell/bootstrappers/watchmaker-bootstrap.ps1

```diff
@@ -87,26 +104,13 @@ function Get-Python {
 
   Download-File $PythonUrl $SavePath
   Install-Python-MSI ${SavePath}
-  Log "Installed Python ${Version}."
+  Log "Installed Python ${PythonVersion}."
 
   [Environment]::SetEnvironmentVariable("Path", "${env:Path};C:\Python27\;C:\Python27\Scripts\", "Machine")
   Log "Added Python directories to environment variable, PATH."
   Reset_Env_Variable
 
-  # Upgrade pip.
-  # pip install --upgrade pip : Do not use this as it throws an access denied error.
-  easy_install -U pip
-  Log "Upgraded pip using easy_intall."
-
-  # Install Watchmaker and Python dependencies for WatchMaker.
-  if( ${WatchmakerUrl} -eq "" ) {
-    Log "The url to a Watchmaker Python egg is empty, unable to install Watchmaker."
-  } else {
-    easy_install ${WatchmakerUrl}
-    Log "Installed Plus3's Watchmaker and dependencies for Python."
-    python -c 'import watchmaker'
-    Log "Tested that Watchmaker installed."
-  }
+  Install-Python-Packages
```

Comment:
How about moving this call out of the `Get-Python` function, and calling it right after the call later to `Get-Python`?